requirements-dev.txt

```diff
@@ -9,3 +9,5 @@ nbconvert==7.4.0
 nbformat==5.8.0
 testcontainers==3.7.1
 qiskit_optimization==0.5.0
+pandas
+matplotlib
```

Comment:
both should be in the classical requirements.txt not here :
https://github.com/IceKhan13/purplecaffeine/blob/3-jupyter-widget-ipywidgets/client/requirements.txt

Reply:
Also miss `ipywidgets` in the requirements.txt as well

Reply:
Also can you put a version (latest if works) :)